libweb3jsonrpc/RPCServer.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include "SafeHttpServer.h"
+#include "ModularServer.h"
+#include "IpcServer.h"
+
+template <class I, class... Is>
+class RPCServer: public ModularServer<I, Is...>
```

Comment:
Yes. It's used in aleths. Thought that it may reduce amount of code written, but you are right, it might be redundant after all.